src/procedures/the-cat-api/types.js

```diff
@@ -0,0 +1,82 @@
+const types = require("../../input-types");
+
+const CAT_BREEDS = {
+  Abyssinian: "abys",
+  Aegean: "aege",
+  AmericanBobtail: "abob",
+  AmericanCurl: "acur",
+  AmericanShorthair: "asho",
+  AmericanWirehair: "awir",
+  ArabianMau: "amau",
+  AustralianMist: "amis",
+  Balinese: "bali",
+  Bambino: "bamb",
+  Bengal: "beng",
```

Comment:
Some of these are apparently causing typos checker issues in CI. We should probably just ignore these files from typos checking ([reference](https://github.com/NetsBlox/services/blob/d548f1d075c47336ac62d307e76133478205621d/_typos.toml#L24-L29)).